actions/DataCreation.php

```diff
@@ -29,34 +29,30 @@
  * @package taoDevTools
  * @subpackage actions
  * @license GPLv2 http://www.opensource.org/licenses/gpl-2.0.php
- *         
+ *
  */
 class DataCreation extends \tao_actions_Main
 {
-
-    public function __construct()
-    {
-        parent::__construct();
```

Comment:
The parent it's calling `$this->setIframeHeaders();` are you sure you don't need that in the actions here?

Reply:
It is called automatically